frontend/pendingTranslations.ftl

```diff
@@ -151,7 +151,7 @@ phone-onboarding-step3-input-placeholder = Enter 6-digit code
 phone-onboarding-step3-button-cta = Confirm my phone number
 phone-onboarding-step3-button-edit = Edit true phone number
 phone-onboarding-step3-button-resend = Resend verification code
-phone-onboarding-step3-error-exipred = Try again—the time limit expired.
+phone-onboarding-step3-error-expired = Try again—the time limit expired.
```

Comment:
Presumably changes to this file require changes elsewhere. I'm unlikely to have the resources to do that.